docs/source/tasks/table_top_gripper/index.md

```diff
@@ -282,3 +282,27 @@ One robot must push/give the cube on its side to the other side so the other rob
 <video preload="auto" controls="True" width="100%">
 <source src="https://github.com/haosulab/ManiSkill/raw/main/figures/environment_demos/TwoRobotStackCube-v1_rt.mp4" type="video/mp4">
 </video>
+
+## RollBall-v1
+![dense-reward][reward-badge]
+
+:::{dropdown} Task Card
+:icon: note
+:color: primary
+
+**Task Description:**
+A simple task where the objective is to push and roll a ball to a goal region at the other end of the table.
+
+**Supported Robots: (Panda, Panda)**
+
+**Randomizations:**
+- the ball's xy position is randomized on top of a table in the region [0.2, 0.5] x [-0.4, 0.7]. It is placed flat on the table
+- the target goal region is marked by a red/white circular target. The position of the target is randomized on top of a table in the region [-0.4, -0.7] x [0.2, -0.9]
+
+**Success Conditions:**
+-  the ball's xy position is within goal_radius (default 0.1) of the target's xy position by euclidean distance.
+:::
+
+<video preload="auto" controls="True" width="100%">
+<source src="https://github.com/haosulab/ManiSkill/raw/main/figures/environment_demos/RollBall-v1.mp4" type="video/mp4">
```

Comment:
I just checked the video. how did you generate it? It looks like the first frame is from another episode (asking about how you generated since this could be a bug with a tool of ours if used our tools)

Reply:
I think I found where bug could be, when I am saving the trajectory from PPO with "--evaluate" its reset params is empty so that could be root cause. Is there a way to prevent this ?

Reply:
Ah so you are straight running the PPO code in the repo. I will check this